Util/channel/src/main/java/io/deephaven/util/channel/LocalFSChannelProvider.java

```diff
@@ -40,6 +37,12 @@ public SeekableByteChannel getReadChannel(@Nullable final SeekableChannelContext
         return FileChannel.open(Path.of(uri), StandardOpenOption.READ);
     }
 
+    @Override
+    public InputStream getInputStream(SeekableByteChannel channel) {
+        // FileChannel is not buffered, need to buffer
+        return new BufferedInputStream(Channels.newInputStream(ReadableByteChannelNoClose.of(channel)));
```

Comment:
Shouldn't we use our own `Channels.newInputStreamNoClose(channel)` to replace `Channels.newInputStream(ReadableByteChannelNoClose.of(channel))`?

Reply:
Yes.